lib/commands.py

```diff
@@ -808,6 +808,7 @@ def add_global_options(parser):
     group.add_argument("-D", "--dir", dest="electron_cash_path", help="electron cash directory")
     group.add_argument("-P", "--portable", action="store_true", dest="portable", default=False, help="Use local 'electrum_data' directory")
     group.add_argument("-w", "--wallet", dest="wallet_path", help="wallet path")
+    group.add_argument("-wp", "--walletpassword", dest="wallet_password", default=None, help="Supply wallet password")
```

Comment:
I personally think it would be better to have `-p` and `--password`, as "wallet" is already implied by `load_wallet`.